plugins/generator-1.19.4/forge-1.19.4/mappings/itemproperties.yaml

```diff
@@ -0,0 +1,6 @@
+damaged: "minecraft:damaged"

```

Comment:
Don't use minecraft: preffix, but instead use mapping system as we do with other elements, where CUSTOM: prefix is added, ideally in JStateLabel#getState and then mapping system has _mcreator_prefix and _mcreator_map_template

Same for 1.19.2

Reply:
That is done for mod elements AFAIK, we don't have an **Item property** MET.

Reply:
Just do instanceof check in JStateLabel#getState and if not built-in, add CUSTOM: to the text. Then define `_mcreator_map_template: "@modid:@name"`

Reply:
This is where having both an interface and an abstract class really makes sense.

Reply:
Agree

Reply:
✔️ Fixed